windows-terminal-quake/Native/FocusTracker.cs

```diff
@@ -40,5 +42,49 @@ public static void FocusGained(Process process)
 				}
 			});
 		}
+
+		// Checks to see if the current window in focus should ignore hotkey
+		public static bool CheckFocusIgnoreHotKey()
+		{
+			var currentWindowHandle = User32.GetForegroundWindow();
+			var allProcesses = Process.GetProcesses();
+			
+			foreach (var window in allProcesses)
+			{
+				try
+				{
+					var mwHandle = window.MainWindowHandle;
+					if (mwHandle == currentWindowHandle)
+					{
+
+						string mainModuleName;
+						try
+						{
+							mainModuleName = window.MainModule.ModuleName;
+						}
+						catch (Exception e)
+						{
+							Log.Error("Unable to determine module name from current focus window");
+							return false;
+						}
+						// This is the current open handles window
+						if (!String.IsNullOrEmpty(mainModuleName))
+						{
+							if(Settings.Instance.IgnoreHotKeyWindows.Any(i => i == mainModuleName))
```

Comment:
I suggest to make this case-invariant, and maybe removing the need for the ".exe"-extension in the settings file.

Reply:
I did think of this, however I was concerned for process's sharing the same name with differing cases. 
I suppose in windows nature, case insensitivity is the way to go.

Can update this soon :)